base/test.jl

```diff
@@ -168,4 +168,20 @@ macro inferred(ex)
     end
 end
 
+#Test equivalence of eigenvectors/singular vectors taking into account possible
```

Comment:
I don't think we provide documentation for unexported functions. Is this useful enough to be exported?